pkg/restapi/issuer/operation/operations.go

```diff
@@ -20,7 +20,7 @@ import (
 
 const (
 	// API endpoints
-	generateInvitationEndpoint = "/didexchange/invitation"
+	generateInvitationEndpoint = "/didcomm/invitation"
```

Comment:
Who calls this endpoint?

I was thinking of going a slightly different route on the RP side: we currently have an endpoint that returns the presentation-definition. I think we should bundle the didcomm invitation in there as well, so the frontend makes only one call.

Reply:
This will be called by issuer.